Source/SwiftLintFrameworkTests/Yaml+SwiftLintTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Yaml+SwiftLintTests.swift
+//  SwiftLint
+//
+//  Created by Scott Hoyt on 12/28/15.
+//  Copyright © 2015 Realm. All rights reserved.
+//
+
+import XCTest
+import Yaml
+@testable import SwiftLintFramework
+
+class YamlSwiftLintTests: XCTestCase {
```

Comment:
D'oh! I didn't read this right. This tests flattening YAML. These tests are fine. I said nothing.